omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/wrapper/openmrs1_8/Module1_8.java

```diff
@@ -0,0 +1,38 @@
+package org.openmrs.module.webservices.rest.web.v1_0.wrapper.openmrs1_8;
+
+import org.openmrs.BaseOpenmrsMetadata;
+import org.openmrs.module.Module;
+
+import java.io.Serializable;
+
+/**
+ * Created by user on 22.05.16.
```

Comment:
@djazayeri  @rkorytkowski 
I have updated PR. I will add tests in the next commit